docs/openssl_build.md

```diff
@@ -0,0 +1,35 @@
+cmake .. -DEXTERNAL_VIV_SDK=`pwd`/../prebuilt-sdk/android_sdk -DCMAKE_TOOLCHAIN_FILE=~/wksp/tools/android-ndk-r23c/build/cmake/android.toolchain.cmake -DLOCAL_BUILD=ON -DLOCAL_OPENSSL=`pwd`/../../openssl -DANDROID_ABI=arm64-v8a -DBUILD_OPENSSL=ON -DCROSS_ANDROID=ON -DCMAKE_INSTALL_PREFIX=`pwd`/install
```

Comment:
install has default dir, whether it will be better if -DCMAKE_INSTALL_PREFIX does not specify in document.
we can start a new line and write: if you want to specify install dir ,use "-DCMAKE_INSTALL_PREFIX=`pwd`/install" to specify